Docs/source/Hardware/Boards.rst

```diff
@@ -13,6 +13,130 @@ the :term:`Shunt Resistor`.
 
     AS8510 Block Diagram 
 
+Overview
+^^^^^^^^
+`GitHub Link <https://github.com/lhr-solar/BPS-AmperesPCB/tree/master>`__
+
+`BOM Link <https://www.mouser.com/ProjectManager/ProjectDetail.aspx?AccessID=b63ac43741>`__ 
+
+Brief Description/Purpose:
+   This board monitors the current and temperature of the BPS system.  
+
+Pertinent Regulations
+^^^^^^^^^^^^^^^^^^^^^
+=========== ============================================== =================================================
+Regulation  Description of Regulation                      How Regulation is Met
+
+**8.3**     | All batteries must have protection circuitry | This board monitors the current/temperature 
+            | appropriate for the battery technology used. | of the battery and sends both measurements 
+            | Proof is required at scrutineering that the  | back to the BPS Leader. Then, the BPS system
+            | protection System is functional and meet     | determines if these measurements are within 
+            | manufacturer's specifications... All         | normal range and if not, will take 
+            | protection circuitry should be contained in  | apropriate action.
+            | the battery enclosure per Reg 8.4. 
+
+**8.3.A.5** | System in which measurements are constantly  | This board sends current and temperature    
+            | monitored and where actions are taken        | measurement back to the BOS leader 
+            | immediately without operator intervention    | periodically. This helps the BPS Leader board 
+            | to open the Main Power Switch should a       | determine when a Battery protection Fault 
+            | battery Protection Fault occur. Any          | occurs and take appropriate actions.
+            | manual clearing process is required by the   
+            | driver with the vehicle not in motion and    
+            | only after faults have been verified clear   
+            | by the protection system.        
+=========== ============================================== =================================================
+
+Context
+^^^^^^^
+**Location of the Board:** Battery Box 
+
+**List of I/O and Connections:**
+    
+    * Power +12 V 
+        * Input from BPS Leader Board 
+    * Power GNDPWR 
+        * Input from BPS Leader Board 
+    * Signal ETS  
+        * Input from Shunt Resistor Board 
```

Comment:
I guess this doesn't matter for other connections (like TX or RX), but it would be helpful to know what these acronyms mean or what they are used for

Reply:
okay will add the acronym or what they are based off the datasheet